add-column.md

```diff
@@ -18,6 +18,104 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
 
 ## Parameters
 
+| Parameter | Description |
+|-----------|-------------|
+| `table_name` | The name of the table to which you want to add the column. |
+| `name` | The name of the column you want to add. The name must be unique to the database and follow these [identifier rules](keywords-and-identifiers.html#identifiers).  |
+| `typename` | [Data type](data-types.html) of the new column. |
+| `col_qualification` | A list of column definitions, which may include [column-level constraint](constraints.html), [collation](collate.html), and/or [column family](column-families.html). |
+
 ## Examples
 
-## See Also
\ No newline at end of file
+### ADD A SINGLE COLUMN
```

Comment:
Quick note: For all these ### example headings, please just use title case, e.g., `Add a Single Column`. That way, these headings won't be in all caps in the TOC at the top of the page.